packages/api/src/utils/db-client.js

```diff
@@ -637,6 +637,20 @@ export class DBClient {
 
     return stats
   }
+
+  /**
+   * Copies upload history and (optionally) user tags and API keys from one
+   * user account to another. Does not modify the original user account or
+   * upload entries.
+   *
+   * @param {import('./db-client-types').CopyUserDataInput} input
+   */
+  async copyUserData(input) {
+    const rsp = await this.client.rpc('copy_user_data', input)
+    if (rsp.error) {
+      throw new DBError(rsp.error)
+    }
+  }
```

Comment:
I think that we should not expose this per https://github.com/nftstorage/nft.storage/pull/1964#issuecomment-1148349005 as this will probably take more than 30s for some users